apps/storybook/.storybook/preview.js

```diff
@@ -36,16 +36,17 @@ export const globalTypes = {
 }
 
 const ThemeProviderDecorator = (Story, context) => {
+  const colorMode = context.parameters?.colorMode || context.globals.colorMode
```

Comment:
Was this needed for this change, or just a random optimisation?

Reply:
This was part of this change

> Unrelated to these changes; updated the Color Mode plugin to allow per-story color mode overrides via the parameters property.

It came about as I was updating the Accordion's stories and spotted one where we were manually simulating our 'all' color mode using grids and a theme provider. I've replaced that with just using our pre-existing color mode ([here](https://github.com/primer/brand/pull/940/files#diff-83a3f0c1c28d8a58ed806a53d5a2febb3ba84069e5747c35bbe1acbaf7bd4245R75))

Happy to remove that if you'd prefer it be pulled out.